admin/timescale/create_extensions.sql

```diff
@@ -1 +1,2 @@
+CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
```

Comment:
I have no idea why this is suddenly required, but the create_hypertable function was failing without it.  :shrug: